pandas/tests/arrays/string_/test_string.py

```diff
@@ -226,6 +226,18 @@ def test_astype_int():
     tm.assert_extension_array_equal(result, expected)
 
 
+@pytest.mark.parametrize(
```

Comment:
can you create any_float_allowed_nullable_dtype in pandas/conftest with numpy float + FLOATING_EA_DTYPE. note that we need to fix the naming of the fixtures as they are confusing,